composer.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "humanmade/analytics",
+    "description": "Analytics module for Platform",
+    "type": "package",
+    "require": {
+        "humanmade/hm-gtm": "1.1.0"
+    },
+    "license": "GPL-3.0",
```

Comment:
I think this should be 2-or-later to match the other modules.

Reply:
Ah ok. I've defaulting to GPL v3 everywhere recently.